less/index.less

```diff
@@ -141,6 +184,12 @@ header {
             color: white;
         }
     } //cta-text 
+    
+    img {
+        @media(max-width: 768px) {
+            display: none;
+        }
+    }
 } //cta
 
 
```

Comment:
You'll want to have only 1 blank line between each section, or you may run into linting errors once you start using linters later on in the curriculum.